### ISSUE TITLE
Is it possible to turn inference parameters off? 

### ISSUE THREAD

**Issue by theadammurphy** [AIStream-Peelout/flow-forecast#363]
I just want to run my model on training and validation sets and get the losses and accuracy scores. Can I set everything to None? It's also unclear how these params relate to my datasets. 

Also, I am actually trying to use FF for a multi-class classification problem with sequence data and not pure time series, so the params `'datetime_start'` and `'hours_to_forecast'` are tripping me up. Should my csv files be set up so that the index is in datetime format? If so, how does that impact `dataset_params.train_end` and `dataset_params.val_start` etc.? Do they also have to be datetime or can they be ints?

**Comment by isaacmg** [comment]
You can import those functions individually if you want. Basically everything you need is in this elif block https://github.com/AIStream-Peelout/flow-forecast/blob/033792cd460f0d1af5f03403309a80d44ea2949b/flood_forecast/trainer.py#L35

At the moment there is not full support for classification, that will come in a future release. If you want to use the inference for classification then you will need to have a datetime index. Also note classification may cause unforeseen bugs,

**Comment by isaacmg** [comment]
Also note classification may cause unforeseen bugs.

**Comment by isaacmg** [comment]
Classification is now officially supported in the latest release  https://github.com/AIStream-Peelout/flow-forecast/releases/tag/flow_al_1.0 . See https://www.kaggle.com/isaacmg/time-series-classification-with-flow-forecast-ff for an example. Thanks for your patience!
